ktxspec.adoc

```diff
@@ -1070,15 +1073,43 @@ have shaders that understand this channel layout.
 The bitstream of the UASTC data is described in Chapter 25 _UASTC
 Compressed Texture Image Format_ of <<KDF13>>.
 
+Basis Universal ASTC HDR Format::
+The Universal ASTC HDR image format (UASTC HDR) is indicated by
+`colorModel` `KHR_DF_MODEL_UASTC_HDR` (= 167). This format supports
+two compression ratios defined by the texel block size in the DFD:
+4x4 and 6x6. Images in this format are a strict RGB-only ASTC HDR
+subset so `vkFormat` must be set to whichever of
+`VK_FORMAT_ASTC_4x4_SFLOAT_BLOCK` (= 1000066000) or
+`VK_FORMAT_ASTC_6x6_SFLOAT_BLOCK` (= 1000066004) matches the texel
+block size. The DFD must be as described in Section ??? of <<KDF13>>.
+Images in this format must be transcoded to BC6H or decoded to raw
+16-bit floating-point values before being uploaded to and sampled
+by a GPU that does not support ASTC HDR.  UASTC HDR images can be
+supercompressed with any scheme except BasisLZ (`supercompressionScheme`
+= 1). If supercompression is used, the DFD must follow the rules
```

Comment:
Ditto.

Reply:
Fixed in https://github.com/KhronosGroup/KTX-Specification/pull/216/commits/1f207fa65dd60a700ee88e6758d1ed67b2a8e577.